generate_documentation.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Actually, I see that this docker image was first build by Phillip, but it seems it is not used in our daily testing of the CGAL documentation.

We could try to run that docker image in parallel to the other process, evaluate the results, and eventually switch to the Docker-based process.

But first Maxime needs to modify the code according to the comments you made two weeks ago.